app/Http/Middleware/Api/Client/Server/AuthenticateServerAccess.php

```diff
@@ -65,17 +63,26 @@ public function handle(Request $request, Closure $next)
             }
         }
 
-        if ($server->suspended && !$request->routeIs('api:client:server.resources')) {
+        if ($server->suspended && ! $request->routeIs('api:client:server.resources')) {
             throw new BadRequestHttpException(
                 'This server is currently suspended and the functionality requested is unavailable.'
             );
         }
 
-        if (! $server->isInstalled()) {
-            // Throw an exception for all server routes; however if the user is an admin and requesting the
-            // server details, don't throw the exception for them.
-            if (! $user->root_admin || ($user->root_admin && ! $request->routeIs($this->except))) {
-                throw new ConflictHttpException('Server has not completed the installation process.');
+        // Still allow users to get information about there server if it is installing or being transferred.
+        if (! $request->routeIs('api:client:server.view')) {
+            if (! $server->isInstalled()) {
+                // Throw an exception for all server routes; however if the user is an admin and requesting the
+                // server details, don't throw the exception for them.
+                if (! $user->root_admin || ($user->root_admin && ! $request->routeIs($this->except))) {
+                    throw new ConflictHttpException('Server has not completed the installation process.');
+                }
+            }
+
+            if ($server->transfer !== null) {
+                if (! $user->root_admin || ($user->root_admin && ! $request->routeIs($this->except))) {
+                    throw new ConflictHttpException('Server is currently being transferred.');
```

Comment:
Can make a `ServerTransferingException` that implements the `HttpExceptionInterface` like some others do so it is a defined exception with a preset message and response code. 👍🏼